benchmarks/Csharp/Benchmarks/Rich/IdleTaskDataRecord.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Harness.Benchmarks.Rich;
```

Comment:
Please don't abbreviate the benchmark name in the folder name.

It's also not clear that you follow the same code style across benchmarks. Wouldn't it be more consistent to either make all classes separate files, or have all benchmarks each in a single file?

Reply:
I had that, but my problem was that C# cannot handle a class and a Sub-Namespace with the same name.
I already had a class "Richards" in the namespace `Harness.Benchmarks` so I renamed the sub namespace. (Note that namespaces need not correlate with folder structure in C#)

As Richards has much more code than nbody, I felt it might need more structure. If you think they should use the same style, which one would you prefer?

Reply:
Yeah, it's more code, but if we can have a single file per benchmark that would match better what we have for other benchmarks. So, it's a bit of an attempt to get cross-language consistency. Helps with maintaining all the languages... especially when bugs are found, which gets tedious with the increasing number of languages :)

So, avoiding variation is a good thing to keep things going.